mxnet/inference/docker/1.9/py3/Dockerfile.cpu

```diff
@@ -49,7 +49,12 @@ ENV PYTHONDONTWRITEBYTECODE=1 \
 ENV MKL_THREADING_LAYER=GNU
 
 RUN apt-get update \
- && apt-get -y upgrade --only-upgrade systemd \
+ && apt-get -y upgrade --only-upgrade \
+    systemd \
```

Comment:
Why are we explicitly listing these packages. Are they not automatically installed as dependencies of other packages?

Reply:
Thanks for catching this. I am removing the packages that I added - they are already being installed in the newest version, so we do not need to add them here.